content/reusable-snippets.mdx

```diff
@@ -3,29 +3,115 @@ title: Reusable Snippets
 description: Reusable, custom snippets to keep content in sync
 ---
 
-One of the core principles of software development is DRY (Don't Repeat Yourself). This is a principle that we should apply to our documentation as well. If you find yourself repeating the same content in multiple places, you should consider creating a custom snippet to keep your content in sync.
+One of the core principles of software development is DRY (Don't Repeat
+Yourself). This is a principle that we should apply to our documentation as
+well. If you find yourself repeating the same content in multiple places, you
+should consider creating a custom snippet to keep your content in sync.
 
-## Creating and using a custom snippet
+## Creating a custom snippet
 
-1. Create a new file in the `_snippets` directory.
-2. Construct your snippet content - it can be a component or a chunk of text.
+1. Create a new file in the `snippets` directory.
 
-```md _snippets/my-snippet.mdx
-## My Snippet
+<Note>
+  Any page in the `snippets` directory will be treated as a snippet and will not
+  be rendered into a standalone page. If you want to create a standalone page
+  from the snippet import the snippet into another file and call it as a
+  component.
+</Note>
+
+### Default export
+
+2. Add content to your snippet file that you want to re-use across multiple
+   locations. Optionally, you can add variables that can be filled in via props
+   when you import the snippet.
+
+```md snippets/my-snippet.mdx
+Hello world! This is my content I want to reuse across pages. My keyword of the
+day is {word}.
+
+## Header 2
 
-This is my snippet.
+`code`
 
-<Info>I want to use this content in more than one place in my docs.</Info>
+<div>I <3 MDX</div>
 ```
 
-3. Navigate to the file where you want to use the snippet.
-4. Add the snippet to the file using the following syntax:
+<Warning>
+  The content that you want to reuse must be inside the `snippets` directory in
+  order for the import to work.
+</Warning>
+
+3. Import the snippet into your destination file.
 
 ```md destination-file.mdx
-<Snippet file="my-snippet.mdx" />
+---
+title: My title
+description: My Description
+---
+
+import MySnippet from '/snippets/path/to/my-snippet.mdx';
+
+## Header
+
+Lorem impsum dolor sit amet.
+
+<MySnippet word="bananas" />
 ```
 
-<Note>
-  You do not need to prepend the "_snippets" directory to the file name and you
-  do not need to import the snippet.
-</Note>
+### Reusable variables
+
+2. Export a variable from your snippet file:
+
+```md snippets/path/to/custom-variables.mdx
```

Comment:
does syntax highlighting get better if you use `mdx` instead of `md`?

Reply:
it does not in mintlify but it does in vscode lol so i changed it